components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java

```diff
@@ -196,7 +196,7 @@ else if (args[i].equals("-tiley")) {
           catch (NumberFormatException e) { }
         }
         else if (!args[i].equals(CommandLineTools.NO_UPGRADE_CHECK)) {
-          LOGGER.error("Found unknown command flag: {}; exiting.", args[i]);
+          System.out.println("Found unknown command flag: " + args[i] + "; exiting.");
```

Comment:
Sure I can look into that. The initial reason for making the change that is logback is not configured for the tests and the default formatter includes a timestamp which was breaking the assert. I'll review how to configure logback for the tests.